internal/dag/httpproxy_processor.go

```diff
@@ -219,8 +224,7 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 				ref := defaultExtensionRef(auth.ExtensionServiceRef)
 
 				if ref.APIVersion != contour_api_v1alpha1.GroupVersion.String() {
-					sw.SetInvalid("Spec.Virtualhost.Authorization.ServiceRef specifies an unsupported resource version %q",
-						auth.ExtensionServiceRef.APIVersion)
+					validCond.AddErrorf("AuthError", "AuthBadResourceVersion", "Spec.Virtualhost.Authorization.extensionRef specifies an unsupported resource version %q", auth.ExtensionServiceRef.APIVersion)
```

Comment:
```suggestion
					validCond.AddErrorf("AuthError", "AuthBadResourceVersion",
					        "Spec.Virtualhost.Authorization.extensionRef specifies an unsupported resource version %q",
					        auth.ExtensionServiceRef.APIVersion)
```

Break super long lines.